pgjdbc/src/test/java/org/postgresql/test/jdbc2/ParameterStatusTest.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2019, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.test.jdbc2;
+
+import org.postgresql.PGConnection;
+import org.postgresql.core.ServerVersion;
+import org.postgresql.test.TestUtil;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.Statement;
+import java.util.Map;
+import java.util.TimeZone;
+import java.util.logging.Logger;
+
+//
+// This test covers client interface for server ParameterStatus messages
+// (GUC_REPORT) parameters via PGConnection.getParameterStatuses() and
+// PGConnection.getParameterStatus() .
+//
+public class ParameterStatusTest extends BaseTest4 {
+
+  private final TimeZone tzPlus0800 = TimeZone.getTimeZone("GMT+8:00");
+  private final Logger logger = Logger.getLogger(ParameterStatusTest.class.getName());
+
+  private Connection con;
```

Comment:
There's a connection in `BaseTest4` why add an extra field?

Reply:
@vlsi I missed that when reading, I'll revise.